### ISSUE TITLE
Fastboot Support

### ISSUE THREAD

**Issue by Aaronfischer** [amiel/ember-data-url-templates#30]
Hello! Curious if it is anywhere in the roadmap to support FastBoot? (We love and have used ember-data-url-templates on just about every project.)

Working on converting my existing ember app to use FastBoot for the first time and I'm getting a `There was an error running your app in fastboot. More info about the error:
 ReferenceError: UriTemplate is not defined`.

References this block:
```js
compileTemplate: function compileTemplate(template) {
  return new UriTemplate(template);
},
```

**Comment by amiel** [comment]
Hi @Aaronfischer.

I don't have any apps using FastBoot, so I don't have much context on how to fix this or a way to test it. It looks like an issue with the way ember-data-url-templates is using bower to load `UriTemplate` though.

I would love for ember-data-url-templates to support FastBoot, and if you have the drive to figure out how to make it so, I would super appreciate a pull-request :)

**Comment by Aaronfischer** [comment]
Hey @amiel!

All good, could you tell me a bit more about `UriTemplate`? Assuming this is an npm dependency that your addon loads/exposes globally?

**Comment by amiel** [comment]
@Aaronfischer Yeah, it gets imported here: https://github.com/amiel/ember-data-url-templates/blob/master/index.js#L8, but this is probably the culprit: https://github.com/amiel/ember-data-url-templates/blob/master/addon/mixins/url-templates.js#L3.
